app/components/badge-maintenance.js

```diff
@@ -0,0 +1,34 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { alias } from '@ember/object/computed';
+
+export default Component.extend({
+    tagName: 'span',
+    classNames: ['badge'],
+    escapedStatus: computed('badge', function() {
+        let regex = /-/g;
+        return this.get('badge.attributes.status').replace(regex, '--');
```

Comment:
Hm, in my opinion, this would actually be clearer if the regex is inline rather than in a variable, so `.replace(/-/g, '--')`. That way, what we're replacing and what we're replacing it with is closer together and easier to read. Can you elaborate on why you have it in a variable here?

Reply:
Because it didn't occur to me I could do it this way too :). The inline approach does look neater, I've changed it to be this way.